### PR TITLE
start German translation of e-commerce

### PR DESCRIPTION
**Pull request by carlotrimarchi**
I started working on a German translation of your project.

So far I translated the first 3 sections

### CODE REVIEW

File: e-commerce-de.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# E-commerce Website
+
+## Willkommen in unserem Geschäft! 🛒😃
+
+### Produktkategorien.
+
+- [ ] Elektronik 📱💻🎮
```
Comment (marcodci) [inline e-commerce-de.md]: Hey, the Elektronik should be toggled

**Review by marcodci — CHANGES_REQUESTED** [review]
Hey, great idea! Have a look to the comments and apply the changes I requested.

File: e-commerce-de.md (hunk 1 above)
Comment (carlotrimarchi) [reply e-commerce-de.md]: I fixed this in this commit: c3d4d36fd197d891a87421feb322b4d6fc217132

**Review by marcodci — APPROVED** [review]
LGTM!

**Pull request merged**